client/sites-dashboard/components/searchable-sites-table.tsx

```diff
@@ -45,7 +45,11 @@ export function SearchableSitesTable( { sites }: SearchableSitesTableProps ) {
 							placeholder={ __( 'Search by name or domain…' ) }
 						/>
 					</div>
-					<SitesTable sites={ filteredSites } />
+					{ filteredSites.length > 0 ? (
+						<SitesTable sites={ filteredSites } />
+					) : (
+						<h2>No sites match your search.</h2>
```

Comment:
I wanted to add this as a state in the `sites-container `component but I think handling only global empty states there make more sense. I would have to create some reference to `searchable-sites `state since it's a child component so I could determine if a search term is entered or not, then re-render the `sites-container` to display the no sites found state.

Reply:
That's ok. Remember to wrap the string with `__` so it gets translated 😉